docs/modules/ROOT/pages/development.adoc

```diff
@@ -378,7 +378,7 @@ Some cops apply changes that only apply in particular contexts, such as if the u
 
 ==== Requiring a minimum Ruby version
 
-If your cop uses new Ruby syntax or standard library APIs, it should only autocorrect if the user has the target Ruby version, which you set with https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/TargetRubyVersion#minimum_target_ruby_version-instance_metho[`TargetRubyVersion#minimum_target_ruby_version`].
+If your cop uses new Ruby syntax or standard library APIs, it should only register offenses if the user has the proper target Ruby version, which you can require with https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/TargetRubyVersion#minimum_target_ruby_version-instance_method[`TargetRubyVersion#minimum_target_ruby_version`].
```

Comment:
Typo in the docs link (missing `d` from `method` at the end.

I also slightly tweaked it. It seems weird to me that it would talk about autocorrect when the cop is just not doing anything at all. It's elaborated on further down but it seems unnecessary